### ISSUE TITLE
rlang warning

### ISSUE THREAD

**Issue by kendonB** [tjmahr/polypoly#1]
``` r
library(polypoly)
df <- data.frame(time = rep(1:5, 3), y = rnorm(15))

# adds columns "time1", "time2", "time3"
poly_add_columns(df, time, degree = 3)
#> Warning: Prefixing `UQS()` with the rlang namespace is deprecated as of rlang 0.3.0.
#> Please use the non-prefixed form or `!!!` instead.
#> 
#>   # Bad:
#>   rlang::expr(mean(rlang::UQS(args)))
#> 
#>   # Ok:
#>   rlang::expr(mean(UQS(args)))
#> 
#>   # Good:
#>   rlang::expr(mean(!!!args))
#> 
#> This warning is displayed once per session.
#>    time           y         time1      time2         time3
#> 1     1  1.04211378 -6.324555e-01  0.5345225 -3.162278e-01
#> 2     2  0.62708025 -3.162278e-01 -0.2672612  6.324555e-01
#> 3     3 -0.56308600 -1.481950e-18 -0.5345225  1.786843e-17
#> 4     4 -1.62341810  3.162278e-01 -0.2672612 -6.324555e-01
#> 5     5 -0.18430943  6.324555e-01  0.5345225  3.162278e-01
#> 6     1  1.60299927 -6.324555e-01  0.5345225 -3.162278e-01
#> 7     2  1.32294537 -3.162278e-01 -0.2672612  6.324555e-01
#> 8     3 -1.35339473 -1.481950e-18 -0.5345225  1.786843e-17
#> 9     4 -0.66145697  3.162278e-01 -0.2672612 -6.324555e-01
#> 10    5 -1.54428737  6.324555e-01  0.5345225  3.162278e-01
#> 11    1 -0.99936272 -6.324555e-01  0.5345225 -3.162278e-01
#> 12    2  0.62082797 -3.162278e-01 -0.2672612  6.324555e-01
#> 13    3  0.32874484 -1.481950e-18 -0.5345225  1.786843e-17
#> 14    4 -1.83552373  3.162278e-01 -0.2672612 -6.324555e-01
#> 15    5  0.01047208  6.324555e-01  0.5345225  3.162278e-01
```

<sup>Created on 2020-01-04 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 3.6.2 (2019-12-12)
#>  os       Ubuntu 18.04.3 LTS          
#>  system   x86_64, linux-gnu           
#>  ui       X11                         
#>  language (EN)                        
#>  collate  en_NZ.UTF-8                 
#>  ctype    en_NZ.UTF-8                 
#>  tz       Pacific/Auckland            
#>  date     2020-01-04                  
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version date       lib source        
#>  assertthat    0.2.1   2019-03-21 [1] CRAN (R 3.6.0)
#>  backports     1.1.5   2019-10-02 [1] CRAN (R 3.6.1)
#>  callr         3.2.0   2019-03-15 [1] CRAN (R 3.6.0)
#>  cli           1.1.0   2019-03-19 [1] CRAN (R 3.6.0)
#>  crayon        1.3.4   2017-09-16 [1] CRAN (R 3.6.0)
#>  desc          1.2.0   2018-05-01 [1] CRAN (R 3.6.1)
#>  devtools      2.1.0   2019-07-06 [1] CRAN (R 3.6.1)
#>  digest        0.6.22  2019-10-21 [1] CRAN (R 3.6.1)
#>  evaluate      0.14    2019-05-28 [1] CRAN (R 3.6.0)
#>  fs            1.3.1   2019-05-06 [1] CRAN (R 3.6.0)
#>  glue          1.3.1   2019-03-12 [1] CRAN (R 3.6.0)
#>  highr         0.8     2019-03-20 [1] CRAN (R 3.6.0)
#>  htmltools     0.3.6   2017-04-28 [1] CRAN (R 3.6.0)
#>  knitr         1.26    2019-11-12 [1] CRAN (R 3.6.1)
#>  magrittr      1.5     2014-11-22 [1] CRAN (R 3.6.0)
#>  memoise       1.1.0   2017-04-21 [1] CRAN (R 3.6.1)
#>  pillar        1.4.2   2019-06-29 [1] CRAN (R 3.6.1)
#>  pkgbuild      1.0.4   2019-08-05 [1] CRAN (R 3.6.1)
#>  pkgconfig     2.0.3   2019-09-22 [1] CRAN (R 3.6.1)
#>  pkgload       1.0.2   2018-10-29 [1] CRAN (R 3.6.1)
#>  polypoly    * 0.0.2   2017-05-27 [1] CRAN (R 3.6.2)
#>  prettyunits   1.0.2   2015-07-13 [1] CRAN (R 3.6.0)
#>  processx      3.3.1   2019-05-08 [1] CRAN (R 3.6.0)
#>  ps            1.3.0   2018-12-21 [1] CRAN (R 3.6.0)
#>  R6            2.4.1   2019-11-12 [1] CRAN (R 3.6.1)
#>  Rcpp          1.0.3   2019-11-08 [1] CRAN (R 3.6.1)
#>  remotes       2.1.0   2019-06-24 [1] CRAN (R 3.6.1)
#>  rlang         0.4.1   2019-10-24 [1] CRAN (R 3.6.1)
#>  rmarkdown     1.13    2019-05-22 [1] CRAN (R 3.6.0)
#>  rprojroot     1.3-2   2018-01-03 [1] CRAN (R 3.6.1)
#>  sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.6.1)
#>  stringi       1.4.3   2019-03-12 [1] CRAN (R 3.6.0)
#>  stringr       1.4.0   2019-02-10 [1] CRAN (R 3.6.0)
#>  testthat      2.2.1   2019-07-25 [1] CRAN (R 3.6.1)
#>  tibble        2.1.3   2019-06-06 [1] CRAN (R 3.6.0)
#>  usethis       1.5.1   2019-07-04 [1] CRAN (R 3.6.1)
#>  withr         2.1.2   2018-03-15 [1] CRAN (R 3.6.0)
#>  xfun          0.11    2019-11-12 [1] CRAN (R 3.6.1)
#>  yaml          2.2.0   2018-07-25 [1] CRAN (R 3.6.0)
#> 
#> [1] /home/kendonb/R/x86_64-pc-linux-gnu-library/3.6
#> [2] /usr/local/lib/R/site-library
#> [3] /usr/lib/R/site-library
#> [4] /usr/lib/R/library
```

</details>

**Comment by tjmahr** [comment]
Thanks for creating the issue

**Comment by mvuorre** [comment]
I've submitted a fix in #3, changing all of 10 characters of code 😄
